applications/tari_console_wallet/src/grpc/mod.rs

```diff
@@ -3,4 +3,54 @@
 
 mod wallet_grpc_server;
 
+use tari_app_grpc::tari_rpc::TransactionEvent;
+use tari_utilities::hex::Hex;
+use tari_wallet::transaction_service::storage::models::{
+    CompletedTransaction,
+    InboundTransaction,
+    OutboundTransaction,
+};
+
 pub use self::wallet_grpc_server::*;
+
+
+pub enum TransactionWrapper {
```

Comment:
Now that we're not going via Notifier, this bridging type is unnecessary. But I'm ok with doing this in a follow-up PR.